docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing
+
+## Overview
+This file contains general contributing information, but individual parts also may have more specific information to them, linked here:
+    - ``[FFBEffects-T300RS](./FFBEffects-T300RS.md)``: Force Feedback Effects example for T300RS
```

Comment:
This doesn't seem to create a link, should the backticks be inside `[]`? Or is it just GitHub being weird with markdown previews?

Reply:
That was a formatting error which was fixed in https://github.com/Kimplul/hid-tmff2/pull/73/commits/fe7751d6c9eb7fcc318691636d08654a75736934#